tests/tools/context.py

```diff
@@ -2,9 +2,11 @@
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
 from dpgen.tools.run_report import *
-
+machine_file = 'machine_fp_single.json'
```

Comment:
Different tests may use different machine_file. I would not recommend define the it in the context.py, which serves to setup the environment and define  functions for global usage.

Reply:
Fixed